### PR TITLE
Remove unusable option session_handler

### PR DESCRIPTION
**Pull request by atrol**
Fixes #23966

### CODE REVIEW

**Comment by atrol** [comment]
> I would only suggest rewording the commit message

I will add the additional information.

> `Remove unused option session_handler`

Not sure if this should be changed.
I thought that unusable is the better word than unused.
The option is used in code, but setting to any other value than the default 'php' does not work, thus it's unusable.
WDYT?

**Comment by dregad** [comment]
Yes you're right

**Comment by Dimas-sc** [comment]
Sorry if it isn't the place to ask it but.. Is it possible to remember the session after browser/PC restart? Every time I've to log in. Thx.

**Comment by atrol** [comment]
> Sorry if it isn't the place to ask it

It isn't, this is the right place to ask such kind of questions https://www.mantisbt.org/forums

To answer your question: It is possible, there is a check box for it on the page where you enter the password.
